packages/dashboard/src/components/pages/trade-type/cfd-test.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import AppToTrade from '../../template/app-to-trade';
+import Instruments from '../../template/instruments';
+import Indices from '../../template/indices';
+import YouMightLike from '../../template/you-might-like';
+import Banner from '../../template/banner';
+
+const card_data = [
+    {
+        title: 'DMT5 Synthetic',
+        sub_title: 'Trade CFDs on synthetic indices that simulate real-world market movements.',
+        icon: 'IcMt5Synthetic',
+    },
+    {
+        title: 'DMT5 Financial',
+        sub_title: 'Trade CFDs on commodities, cryptocurrencies, and currency pairs with leverage.',
+        icon: 'IcMt5FinancialStp',
+    },
+    {
+        title: 'DMT5 Financial',
+        sub_title: 'Trade CFDs on commodities, cryptocurrencies, and currency pairs with leverage.',
+        icon: 'IcMt5FinancialStp',
+    },
+];
+
+const benefit_data = [
+    { icon: 'IcMt5FinancialStp', description: 'High leverage, low spreads' },
```

Comment:
We should localize description for these items as well

Reply:
same as above